### PR TITLE
Make Color Picker divider visible in Clearlooks, Leech

### LINKED ISSUE #102: Clearlooks has no visible #ColorPicker divider
I noticed there is no visible QFrame divider from the Color Picker widget, but of the 12 themes, only in Clearlooks.

Setting `LXQtPanel #BackgroundWidget { background-color: green; }` results in:

image

However, setting `#ColorPicker QFrame { background: red; }` shows that there is a bounding box for it still:

image2

### PR DESCRIPTION
**Pull request by isf63**
Set "border: 0px;" only for required widgets.

Closes #102

### CODE REVIEW

**Comment by isf63** [comment]
@stefonarch, would you mind logging into X11 to test this? ;)

**Comment by stefonarch** [comment]
Oops, I missed this. It works fine, colorpicker is visible also under Wayland, is just doesn't work ;).

@gfgit had some project for it, if I remember well.

**Comment by isf63** [comment]
Looking at https://wayland.app/, it seems the Wlroots protocol is https://wayland.app/protocols/wlr-screencopy-unstable-v1

**Comment by tsujan** [comment]
Currently, the backend system is for taskbar and switcher (the latter working only with the kwin and X11 backends), i.e., for knowing about windows, their states, workspaces, and so on.

File: themes/Clearlooks/lxqt-panel.qss [hunk 1]
```diff
@@ -9,16 +9,15 @@ LXQtPanel #BackgroundWidget {
     border: 0px solid #1a1a1a;
 }
 
-Plugin > QWidget,
-Plugin > QWidget > QWidget {
+Plugin > QWidget/*,
```
Comment (stefonarch) [inline themes/Clearlooks/lxqt-panel.qss:12]: This looks strange to time though.

Comment (tsujan) [reply themes/Clearlooks/lxqt-panel.qss:12]: > it's commented out

Still it's better to remove it.

**Comment by isf63** [comment]
Done.

**Pull request merged**